prime-router/metadata/fhir_mapping/hl7/datatype/CodeableConcept.yml

```diff
@@ -63,7 +63,20 @@ coding_6:
   constants:
     codeIndex: "alternate identifier"
 
+# Allows alternate coding to be added from variables
 coding_7:
+  generateList: true
```

Comment:
Slight re-arrangement and updated the naming here to be less confusing.  This allows to directly set the the alternate identifier in a Codable concept